doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,618 @@
+.. _ncs_release_notes_110:
+
+|NCS| v1.1.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
+   |NCS| v1.1.0 does not provide support for nRF5340.
+   For nRF5340 support, check out the master branch.
+
+Highlights
+**********
+
```

Comment:
Need content here.

Reply:
I would highlight support for modem FOTA.

Reply:
New zephyr driver for CC310 HW initialization and entropy
nrf_security: Passing NIST CAVP tests for nRF9160 and nRF52840 using CryptoCell CC310 + SW
nrf_security: Passing handshake of all major suites on TLS and DTLS using CryptoCell CC310 + SW